inputs/rt.in

```diff
@@ -0,0 +1,69 @@
+<comment>
+problem = Rayleigh-Taylor instability
+
+<job>
+problem_id = rt
+
+<problem/rt>
+iprob  = 1
+amp   = 0.01
+drat  = 2.0
+
+<parthenon/mesh>
+refinement = adaptive
+numlevel = 3
+nghost = 4
+
+nx1       = 100         # Number of zones in X1-direction
+x1min     = -0.16666667 # minimum value of X1
+x1max     = 0.16666667  # maximum value of X1
+ix1_bc    = periodic    # inner-X1 boundary flag
+ox1_bc    = periodic    # outer-X1 boundary flag
+
+nx2       = 200         # Number of zones in X2-direction
+x2min     = -0.5        # minimum value of X2
+x2max     = 0.5         # maximum value of X2
+ix2_bc    = reflecting     # inner-X2 boundary flag
+ox2_bc    = reflecting     # outer-X2 boundary flag
```

Comment:
FYI reflecting boundary conditions are not working and need to be merged, see #58
Realistically, this should happen later this week or early next week.

Reply:
Changes to outflow for now